dask_expr/collection.py

```diff
@@ -126,8 +126,8 @@ def __dask_keys__(self):
         out = out.simplify()
         return out.__dask_keys__()
 
-    def simplify(self):
-        return new_collection(self.expr.simplify())
+    def simplify(self, **kwargs):
```

Comment:
Can we get rid of the kwargs here?

Reply:
Yeah, I didn't want to set a `phases` default in two places, but we can use a `phases=None` default to avoid this.